readme.md

```diff
@@ -38,35 +38,35 @@ Once installation is working, take a look at the existing code to make sure you
 
 * **Question:** Describe what this code is doing and what its purpose is.
 
-* **Your Answer:** 
+* **Your Answer:** Connects to MongoDB and then deletes all Party objects of the database.
 
 ---
 
 - [ ] Imagine that as a user, you enter your username and password into a site in order to signup.
 
 * **Question:** What happens next?
 
-* **Your Answer:**
+* **Your Answer:** It compares the username and password to what is saved in the database and blocks the user if it doesn't match 
```

Comment:
In a typical process, should do validation checks, check if unique, encrypt and save to db.